Release notes for March 2026 (#777)
commerce-docs/microsite-commerce-storefront: src/content/docs/releases/changelog.mdx

************************************/}

      <ChangelogEntry
          date="2026-03-19"
          title="Storefront Compatibility Package v4.8.17"
          components={['Storefront Compatibility Package']}
      >
          The Storefront Compatibility Package has been updated to include the following changes:

>           - **PHP 8.5 support**: Added PHP 8.5 support across all modules of the Storefront Compatibility Package.

Reviewer: ```suggestion
          - **PHP 8.5 support**: Added PHP 8.5 support to all modules in the Storefront Compatibility Package.
```

************************************/}

      <ChangelogEntry
          date="2026-03-19"
          title="Storefront Compatibility B2B Package v1.0.18"
          components={['Storefront Compatibility B2B Package']}
      >
          The Storefront Compatibility B2B Package has been updated to include the following changes:

>           - **PHP 8.5 support**: Added PHP 8.5 support across all modules of the Storefront Compatibility B2B Package.

Reviewer: ```suggestion
          - **PHP 8.5 support**: Added PHP 8.5 support to all modules in the Storefront Compatibility B2B Package.
```

**Wishlist** 
************/}

    <ChangelogEntry
      date="2026-03-16"
      title="Wishlist v3.2.0"
      components={['Wishlist']}
    >
      The Wishlist drop-in has been updated with the following changes:

>       - **Multi-store wishlist** - The drop-in now supports multi-store scenarios. Pass an optional `storeCode` config prop during initialization (the boilerplate reads it from the AEM config system and passes it into the drop-in). When `storeCode` is set and not `'default'`, localStorage and sessionStorage keys and wishlist ID cookies are scoped per store (for example `DROPIN__WISHLIST__WISHLIST__DATA__` plus the store code), so guest and authenticated wishlist data stay isolated between store views. Single-store setups and the `'default'` store continue using the original unscoped keys for backward compatibility ([#73](https://github.com/adobe-commerce/storefront-wishlist/pull/73)). Boilerplate integration ([#1139](https://github.com/hlxsites/aem-boilerplate-commerce/pull/1139)).

Reviewer 2: >the boilerplate reads it from the AEM config system and passes it into the drop-in

What does this mean? Is it documented elsewhere, in wishlist block docs? What is "AEM Config system"?
